common/sai_dut.py

```diff
@@ -131,7 +131,7 @@ def init(self):
         self._assert_redis_is_available()
 
         # Flush SONiC Redis content
-        r = redis.Redis(host=self.server_ip, port=self.port, db=1)
+        r = redis.Redis(host=self.server_ip, port=self.port, db=self.asic_db)
```

Comment:
If `asic_db` defined somewhere for this file? I guess no... please revert this change and in `cleanup()`.

Reply:
Ok, i agree. Fixed. Thanks!